Firmata.h

```diff
@@ -15,45 +15,12 @@
 #define Firmata_h
 
 #include "Boards.h"  /* Hardware Abstraction Layer + Wiring/Arduino */
+#include "FirmataConstants.h"
+#include "FirmataMarshaller.h"
 #include "FirmataParser.h"
 
-/* Version numbers for the Firmata library.
- * The firmware version will not always equal the protocol version going forward.
- * Query using the REPORT_FIRMWARE message.
- */
-#define FIRMATA_FIRMWARE_MAJOR_VERSION  2
-#define FIRMATA_FIRMWARE_MINOR_VERSION  5
-#define FIRMATA_FIRMWARE_BUGFIX_VERSION 4
-
-/* DEPRECATED as of Firmata v2.5.1. As of 2.5.1 there are separate version numbers for
- * the protocol version and the firmware version.
- */
-#define FIRMATA_MAJOR_VERSION           2 // same as FIRMATA_PROTOCOL_MAJOR_VERSION
```

Comment:
These 3 deprecated defines need to be included in Firmata.h or older Firmata sketches will break.